src/hooks/useDailyStatistics/useDailyStatistics.tsx

```diff
@@ -83,6 +87,7 @@ export const useDailyStatistics = (
     sessionToken,
     currentTimestamp,
     prevTimestamp,
+    c13ntForUnit,
```

Comment:
I recall mentioning this previously, why does this need to be exported?

Reply:
need it for the screen to translate the uni

Reply:
hmm, I looked through the current repo, we dont import this useTranslate in other hooks. We currently import it in components that needs it haha. 

Do refer to other components, like NoQuotaCard, Stepper, ItemMaxUnitLabel and CheckoutSuccessCard to see how it is being implemented so far haha.

Reply:
Do you mind sharing which screens needs this also haha, thanks!

Reply:
@shermainelim have reopened this, as I have questions on this too haha

Reply:
hmm, i dont think we should import a hook inside another hook. all hooks should only be imported at the component level and passed down if needed haha.

if you noticed in my previous comment, all current implementations follow this format. You can refer to NoQuotaCard, Stepper, ItemMaxUnitLabel and CheckoutSuccessCard to take a look.

Reply:
I think you can call the `useTranslate` hook at the DailyStatistics components, like how it's done now in the TitleStatistic and TransactionHistoryCard

Reply:
-Refactored the typing in useDailyStatistics Utils by adding a new Transaction type that can be used by other files so there is no need to always retype the same typing that was used in the old approach

- Formatting quantity and unit with c13forUnit in TransactionHistoryCard with formatText method. 

- Removed Jest as no longer need to mock the translation as it will be done in the TransactionHistoryCard instead of utils.

Reply:
Thanks haha. I left some comments on the refactoring and changes. Actually for this comment, since we're moving it out of the useDailyStatistics hook, can we remove this line?